challenges/classes.js

```diff
@@ -1,7 +1,23 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
-
-// Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
-
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+class CuboidMaker {
+    constructor(props) {
+    this.length = props.length;
+    this.width = props.width;
+    this.height = props.height;
+  }
+    volume() {
+       console.log(`${this.length * this.width * this.height}`);
```

Comment:
Missing a return statement in both methods. This is why lines 21 and 22 don't work.